package/yast2-migration.changes

```diff
@@ -1,4 +1,11 @@
 -------------------------------------------------------------------
+Thu Sep 24 19:12:38 UTC 2015 - lslezak@suse.cz
+
+- do not display the "success" dialog when the package installation
+  is aborted
```

Comment:
Oh, good point, I wrote the bug number to the commit message, but forgot to write it also to the changes...